components/BridgeProgress.tsx

```diff
@@ -20,6 +20,17 @@ const BridgeProgress: VFC<IntrinsicElements["div"] & BridgeProgressProps> = (
 	const dismissible =
 		txStatus?.status === "Success" || txStatus?.status === "Failure";
 
+	useEffect(() => {
+		const beforeUnload = (event: BeforeUnloadEvent) => {
+			event.preventDefault();
+			if (txStatus?.status === "Pending") return (event.returnValue = "");
```

Comment:
set the `returnValue` to be "There is pending transaction. Are you sure you want to leave?"

That should pop up the message as above when you try to close the browser

Reply:
done, but I still just get "Changes that you made may not be saved." in popup

Reply:
<img width="722" alt="Screen Shot 2022-04-22 at 7 51 54 AM" src="https://user-images.githubusercontent.com/52016903/164541714-1be0c00b-7266-43f8-8b34-dcd95a3a3356.png">
https://stackoverflow.com/questions/38879742/is-it-possible-to-display-a-custom-message-in-the-beforeunload-popup

Reply:
sweet